Client/Frontend/AuthenticationManager/AppAuthenticator.swift

```diff
@@ -42,4 +43,15 @@ class AppAuthenticator {
     static func canAuthenticateDeviceOwner() -> Bool {
         return LAContext().canEvaluatePolicy(.deviceOwnerAuthentication, error: nil)
     }
+
+    static func getDefaultErrorMessageForContext(context: LAContext) -> String {
```

Comment:
We've tried this approach before and it's led to the ticket now. The reason we can't do it this way is because when biometric authentication fails, the description can't be updated (see the last screenshot in the github issue). 

What we actually need is a more generalized string that can suffice as a description for both when a user is attempting biometric auth, and when that fails and asked to put a passcode.

Reply:
Interesting insight, so should I put something like "Secure access requires biometric authentication. If unsuccessful, please provide your passcode when prompted." to cover all the cases or ask someone from Design/UX to help with the string?